### DISCUSSION TITLE
VSCode synctex is not working.

### DISCUSSION THREAD

**Discussion by ini17** [James-Yu/LaTeX-Workshop#3448] [Q&A]
I'm using VSCode + latex-workshop + sumatraPDF.
And my VSCode settings.json is below:
```
"latex-workshop.view.pdf.external.viewer.command": "B:\\0_Myfiles\\Softwares\\SumatraPDF\\SumatraPDF.exe",
"latex-workshop.view.pdf.external.viewer.args": [
    "-forward-search",
    "%TEX%",
    "%LINE%",
    "-reuse-instance",
    "-inverse-search",
    "code.cmd -r -g \"%f:%l\"",
    "%PDF%"
],
"latex-workshop.view.pdf.external.synctex.command": "B:\\0_Myfiles\\Softwares\\SumatraPDF\\SumatraPDF.exe",
"latex-workshop.view.pdf.external.synctex.args": [
    "-forward-search",
    "%TEX%",
    "%LINE%",
    "-reuse-instance",
    "-inverse-search",
    "code \"B:/0_Myfiles/Softwares/Microsoft VS Code/resources/app/out/cli.js\" -r -g "%f:%l\"",
    "%PDF%"
],
```
In the "synctex.args", I tried `code` or `code.cmd` or `PathTOVScode/cmd.exe` and so on.
And in the SumatraPDF settings.txt, I tried all possible methods too.
Actually, I tried so many methods I found from github or VSCode wiki and so on.
**The synctex has not worked even once, but it can just open the correct PDF.**
I guess the problem lies in that my .pdf file is exported to the directory above my .tex file.
Its like:
folder1
┣━━ tex_folder
┣━━━━ a.tex
┗━━ output.pdf
However, I tested the %TEX% and %PDF%. They correctly denote path and file, so I'm very comfused.
I guess whether the %f is not suitable in my situation, but I don't know how to correct it.
I will sincerely appreciate your help, thanks!

**Comment by James-Yu** [comment]
External viewer is experimental and does not have official support.

**Reply by ini17** [reply]
Thanks for your reply!
Do you mean that its no longer in use?
Or just for me ;) or my bad settings.
But my synctex in tab-viewer doesn't work too...
I'm so sad...
